doc/readme.md

```diff
@@ -0,0 +1,16 @@
+# Trapperkeeper Authorization
```

Comment:
@nathanielksmith I don't have a big problem with having the separate file.  I'm probably just used to not looking for a readme.md file in the "doc" folder since there's already one in the base directory.  Might be good to get someone else's opinion on this - especially someone with little/no prior experience with the tk-auth documentation.  Ultimately, if you want to keep this file in, though, I'm good with it.